### PR TITLE
feat(ads): integrate Broadstreet into the providers wizard

### PR DESCRIPTION
**Pull request by miguelpeixe**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Removes WordAds and AdSense integration from Setup and Ads wizards.

image

image

Closes https://github.com/Automattic/newspack-ads/issues/249.

### How to test the changes in this Pull Request:

1. Visit Advertising home wizard and WordAds and AdSense should no longer be available as options
2. Start a new site, complete the onboarding setup and confirm the Google AdSense was not an available "extra feature"

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by adekbadek** [comment]
Now that it's just one provider, this screen does not make much sense. The tab should rather be called "Google Ad Manager" and the view visible after clicking "Configure" should be the content of this tab. Also, it now makes no sense to "turn on" GAM as a provider, since this is the only option.  

image

cc @thomasguillot

**Comment by miguelpeixe** [comment]
Good call! 11663b27e937c9834db5dd82094196371173970c removes the "Ad Providers" screen and makes GAM Ad Units the home of the wizard. I decided to title it "Google Ad Manager" because of the coming plans with [Ads Milestone 4](https://github.com/Automattic/newspack-ads/milestone/4) and further GAM integration.

**Comment by thomasguillot** [comment]
Changes look good, thank you @miguelpeixe

**Comment by miguelpeixe** [comment]
I realized that there are functionalities tied to the toggling of the GAM service. This means that even though we do not have other providers, GAM should still be able to be activated and deactivated.

4324f8ddde01b3db591009f1b64a41e294d626d5 restores the `<ActionCard />` but keep the rest of GAM configuration on the same screen.

image

image

**Comment by adekbadek** [comment]
Option 1 – I like @thomasguillot 's suggestion, I'd only move the the "Google Ad Manager" tab to the last position, since it should be the least used tab. With GAM deactivated, it should be the only tab (which means there can be no tabs at all).

Option 2 – That said, I think @miguelpeixe 's solution is fine, with the first tab name changed to "Ad Units", since that's the main purpose of it. And the action card with the toggle could be moved to "Settings" tab, since this toggle should be used once in most cases. 

I'd say let's do option 1, but if that'd take too much dev resources, option 2 + an "enhancement" issue on GH for option 1 is fine. WDYT?

**Comment by miguelpeixe** [comment]
cc @kmwilkerson, in case you have ideas as well 😃

**Comment by thomasguillot** [comment]
Here's what we could do:

- The tab could be called "Ad Dashboard" or whatever is best
- On init, you'd have the GAM toggle disabled. (unless it was already enabled during the onboarding)
- Once you enable the toggle, the "Ad Dashboard" would display "Ad Units", "Order & Line Items", etc... and then the GAM toggle, alongside the network code would move to the "Settings".

**Comment by miguelpeixe** [comment]
> Once you enable the toggle, the "Ad Dashboard" would display "Ad Units", "Order & Line Items", etc...

Would we use a `<Card />` to link to the ad units? We won't have orders and line items just yet.

**Comment by thomasguillot** [comment]
`buttobcard` like the dashboard wizard

**Comment by miguelpeixe** [comment]
That also means the ads wizard home wouldn't be consistent. What would we put as home in case GAM is deactivated? 🤔

**Comment by miguelpeixe** [comment]
Tagging @adekbadek @thomasguillot, since this has been on hold for a while and is ready for review again

**Comment by thomasguillot** [comment]
> Tagging @adekbadek @thomasguillot, since this has been on hold for a while and is ready for review again

Looking good. I moved GAM's `ActionCard` to be at the top. https://github.com/Automattic/newspack-plugin/pull/1465/commits/c4e3302b43967c7566349fc49b572521e98bb28b

**Review by dkoo — CHANGES_REQUESTED** [review]
Functionally, looks 👍 to me. A few nitpicks below:

File: assets/wizards/advertising/views/ad-unit/index.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ class AdUnit extends Component {
 					>
 						{ __( 'Save', 'newspack' ) }
 					</Button>
-					<Button isSecondary href={ `#/${ service }` }>
+					<Button isSecondary href="#/">
```
[the remark this answers is not in the document]
Comment (miguelpeixe) [reply assets/wizards/advertising/views/ad-unit/index.js]: Nice catch! While testing different flows I changed it and forgot to change it back. aad971cc764ba43541aa0f1725a6f3b8e94003ef fixes it.

File: assets/wizards/advertising/index.js [hunk 2]
```diff
@@ -138,7 +136,7 @@ class AdvertisingWizard extends Component {
 		const { services, adUnits } = advertisingData;
 		const tabs = [
 			{
-				label: __( 'Ad Providers', 'newspack' ),
+				label: __( 'Providers', 'newspack' ),
```
[the remark this answers is not in the document]
Comment (miguelpeixe) [reply assets/wizards/advertising/index.js:139]: I think "Back" is enough. Done in e793f74a6ca9b9a5c0566286d36403d72ad6d077

**Review by dkoo — APPROVED** [review]
Thanks for the changes—looking good to me now!

**Pull request merged**